mev_inspect/aave_liquidations.py

```diff
@@ -14,7 +14,7 @@
 
 def get_aave_liquidations(
     traces: List[ClassifiedTrace],
-) -> List[Liquidation]:
+) -> Optional[List[Liquidation]]:
```

Comment:
i don't see anywhere where the list itself can be none - should just be empty list if we find none